patroni/api.py

```diff
@@ -848,7 +1403,15 @@ def __has_dual_stack():
                     sock.close()
         return False
 
-    def __httpserver_init(self, host, port):
+    def __httpserver_init(self, host: Optional[str], port: int) -> None:
```

Comment:
The host can't be None here, but pyright will probably complain because later it could be assigned to None. It could be solved by introducing a new variable.
```suggestion
    def __httpserver_init(self, host: str, port: int) -> None:
```